letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -545,8 +573,15 @@ if [ "$1" = "--le-auto-phase2" ]; then
     # If the selected Bootstrap function isn't a noop and it differs from the
     # previously used version
     if [ -n "$BOOTSTRAP_VERSION" -a "$BOOTSTRAP_VERSION" != "$PREV_BOOTSTRAP_VERSION" ]; then
-      # if non-interactive mode or stdin and stdout are connected to a terminal
-      if [ \( "$NONINTERACTIVE" = 1 \) -o \( \( -t 0 \) -a \( -t 1 \) \) ]; then
+      # Check if we can rebootstrap without manual user intervention: this require that
```

Comment:
Same as https://github.com/certbot/certbot/pull/7268#discussion_r335715450 and https://github.com/certbot/certbot/pull/7268#discussion_r337692588.

```suggestion
      # Check if we can rebootstrap without manual user intervention: this requires that
```